lib/foreman/fips.rb

```diff
@@ -0,0 +1,16 @@
+module Foreman
+  module Fips
+    def self.md5_available?
+      return @md5_available if defined?(@md5_available)
+      @md5_available = begin
+        require 'openssl'
+        OpenSSL::Digest::MD5.digest('')
+        true
+      rescue LoadError
+        true
```

Comment:
Is this true?

Reply:
I don't know. It definitely doesn't make sense. I guess that happens, when you blindly copy-and-paste code.